packages/trie/src/baseTrie.ts

```diff
@@ -104,7 +104,7 @@ export class Trie {
    */
   async checkRoot(root: Buffer): Promise<boolean> {
     try {
-      const value = await this._lookupNode(root)
+      const value = await this.lookupNode(root)
```

Comment:
We didn't plan to do a breaking Trie release initially, but my previously somewhat stronger opinion on this is wayning, so we might just do along (and then also finally do the renaming).

Just wanted to mention here. We then eventually want to have one additional thought round or broader look on other breaking changes.

Reply:
So, some additional context: currently Trie is still listed in the "Future Release Planning" issue: https://github.com/ethereumjs/ethereumjs-monorepo/issues/1718. Should I move over? 🤔

Reply:
ah sorry didn't notice that.

yeah sure let's include these refactorings, i think now is the time to do it while we are preparing develop. not sure if it makes sense to wait any longer. we can assign some team members to the tasks and i can also dedicate some time this month to tackle them.